docs/setup/embedding.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[embedding]]
+== Embedding {kib}
```

Comment:
Embedding {kib} content in a web page?

Reply:
This page looks very useful! Why is it organized as part of setup instead of being part of dashboards?

Reply:
> Embedding {kib} content in a web page?

Yeah, sounds good, thanks.

> This page looks very useful! Why is it organized as part of setup instead of being part of dashboards?

Basically setup was the only existing top-level section that can potentially cover all parts of Kibana. The idea was to eventually describe everything that can be embedded in one way or another: dashboards, visualizations, canvas, or any random part of Kibana that can technically be embedded (we know people do this).

Reply:
We are planning on creating a Share section in the documentation.  Would this doc be a good candidate for that?

Reply:
Yes, I think so.